pkgs/development/libraries/avro-c++/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, cmake, boost155, pythonPackages
+}:
+
+let version = "1.7.5"; in
```

Comment:
Yay, I know. I just looked through sample packages in nixpkgs manual and found next style is more common:

```nix
let version = "4.2"; in

stdenv.mkDerivation {
  name = "foo-${version}";
}
```